data/csv_importer.py

```diff
@@ -0,0 +1,204 @@
+import csv
+import logging
+import os
+
+from django.db.models import ForeignKey, TextField, CharField, FloatField
+
+
+# Import the model
+from .models.ingredient import Ingredient, IngredientSynonym
+from .models.microorganism import Microorganism
+from .models.plant import Plant, PlantPart, PlantSynonym, PlantFamily
+from .models.population import Population
+from .models.substance import Substance, SubstanceSynonym
+
+logger = logging.getLogger(__name__)
+
+# Modèles pour recevoir l'import des données des fichier csv
+CSV_TO_MODEL_MAPPING = {
+    "REF_ICA_INGREDIENT_AUTRE.csv": Ingredient,
+    "REF_ICA_MICRO_ORGANISME.csv": Microorganism,
+    "REF_ICA_PARTIE_PLANTE.csv": PlantPart,
+    "REF_ICA_PLANTE.csv": Plant,
+    "REF_ICA_SUBSTANCE_ACTIVE.csv": Substance,
+    "REF_ICA_INGREDIENT_AUTRE_SYNONYME.csv": IngredientSynonym,
+    "REF_ICA_PLANTE_SYNONYME.csv": PlantSynonym,
+    "REF_ICA_SUBSTANCE_ACTIVE_SYNONYME.csv": SubstanceSynonym,
+    "POPULATION.CSV": Population,
```

Comment:
J'imagine qu'on préfère que l'extension soit en minuscule ? 
```suggestion
    "POPULATION.csv": Population,
```
D'ailleurs je n'ai pas trouvé un fichier avec ce nom, j'imagine que c'est `unknown_name_file.csv` dans le commit ?